ngi_pipeline/engines/piper_ngi/utils.py

```diff
@@ -170,15 +170,15 @@ def find_previous_sample_analyses(project_obj, sample_obj=None, include_genotype
         if sample_obj and sample.name != sample_obj.name:
             continue
         sample_files = glob.glob(os.path.join(project_dir_pattern,
-                                              "{}\.*".format(sample.name)))
+                                              "{}.*".format(sample.name)))
         # P123_456 is renamed by Piper to P123-456? Sometimes? Always?
-        piper_sample_name = sample.name.replace("_", "-", 1)
+        piper_sample_name = sample.name.replace("_", "?", 1)
```

Comment:
Well, now taca enforces the underscore supremacy as long as it can, but if piper still changes things, I need that to be working, so I give it the benefit of the doubt :P